### ISSUE TITLE
Why do you need my history?

### ISSUE THREAD

**Issue by vitaly-zdanevich** [SleekPanther/youtube-chronological-order#1]
![image](https://user-images.githubusercontent.com/3514015/52909943-50e7c200-32a1-11e9-820b-b4043a7809ad.png)

**Comment by SleekPanther** [comment]
I don't, but in order to have access to certain things like navigating to a new URL and extracting the channel from the current URL I had to declare my extension needs access to certain permissions. I used the minimal set of permissions to get it working and without these permissions my extension simply doesn't function when I try to invoke the chrome API to navigate to a new page.
It said "Read your browsing history" but that's a bit misleading as a categorization. I need the tabs permission to get the URL and navigate to a new one.
I need declarativeContent to make sure my extension is only visible on YouTube video watch pages.
And I need activeTab for things like the keyboard shortcuts, but saying active tab is actually more limiting that it sounds. [https://developer.chrome.com/extensions/activeTab](https://developer.chrome.com/extensions/activeTab)
I hope that clears it up.
